### PR TITLE
Fix: Settings Fallback

### PR DESCRIPTION
**Pull request by park012241**
Add Fallbacks for turnoff-namuwiki

### CODE REVIEW

File: src/popup.ts [hunk 1]
```diff
@@ -66,32 +47,20 @@ const chkbox = [
...
 
 function setHook() {
     for (const chk of chkbox) {
-        console.log(chk);
         const datasetVal = chk.dataset.val as keyof ConfigInterface;
         if (chk.type === "checkbox") {
             chk.checked = config[datasetVal] as boolean;
         } else if (chk.type === "text" || chk.type === "url") {
             chk.value = config[datasetVal] as string;
+            chk.value = chk.value === 'undefined' ? '' : chk.value;
```
Comment (Alex4386) [inline src/popup.ts]: Isn't this should be `typeof chk.value === 'undefined'` ?

Comment (Alex4386) [reply src/popup.ts]: and could you open and close Parentheses `()` for better readability?

Comment (park012241) [reply src/popup.ts]: How About
```TS
chk.value = typeof(config[datasetVal]) !== 'undefined' ? config[datasetVal] as string : '';
```?

**Comment by park012241** [comment]
Please checkout this.

File: src/popup.ts (hunk 1 above)
Comment (Alex4386) [reply src/popup.ts]: Good

**Pull request merged**